tests/fixtures/commands/component-check.yml

```diff
@@ -69,20 +69,27 @@
         >  Composer validation check passed.
 
 
+        Checking Configuration components.
+        ==================================
+
+        >  Project configuration validation check passed.
+
+
         Results:
         ========
 
-        --------------------------- --------
-         Mandatory module check      failed
-         Recommended module check    passed
-         Insecure module check       passed
-         Outdated module check       passed
-         Abandoned module check      passed
-         Unsupported module check    passed
-         Evaluation module check     failed
-         Development module check    passed
-         Composer validation check   passed
-        --------------------------- --------
+        ---------------------------------------- --------
+         Mandatory module check                   failed
+         Recommended module check                 passed
+         Insecure module check                    passed
+         Outdated module check                    passed
+         Abandoned module check                   passed
+         Unsupported module check                 passed
+         Evaluation module check                  failed
+         Development module check                 passed
+         Composer validation check                passed
+         Project configuration validation check   passed
```

Comment:
I am wondering, can we short this label ?
Something like `Project configuration check`